pkg/util/replicated.go

```diff
@@ -36,6 +37,15 @@ func GetReplicatedDeploymentName() string {
 	return "replicated"
 }
 
+func GetReplicatedDeploymentUID(clientset kubernetes.Interface, namespace string) (apimachinerytypes.UID, error) {
```

Comment:
suggestion: let's use this here too? https://github.com/replicatedhq/replicated-sdk/blob/f138052bb486aa32238f142a25f711bbbe99d020/pkg/util/replicated.go#L49-L54